python/ray/actor.py

```diff
@@ -949,9 +949,10 @@ def __dir__(self):
         return self._ray_method_signatures.keys()
 
     def __repr__(self):
-        return (f"Actor("
-                f"{self._ray_actor_creation_function_descriptor.class_name},"
-                f"{self._actor_id.hex()})")
+        return ("Actor("
+                "class_name="
+                f"{self._ray_actor_creation_function_descriptor.class_name}, "
+                f"actor_id={self._actor_id.hex()}")
```

Comment:
question: when a user does:

```
@ray.remote
class Actor:
    def __repr__(self):
        return "hi"
```
does this get overriden?

Reply:
It should, but let me test it out tmrw

Reply:
This should work now. I added a test as well. One caveat is the default is the default repr